tests/providers/apache/hive/transfers/test_mysql_to_hive.py

```diff
@@ -152,172 +96,115 @@ def setUp(self):
                 for row in rows:
                     cur.execute("INSERT INTO baby_names VALUES(%s, %s, %s, %s);", row)
 
-    def tearDown(self):
+                conn.commit()
```

Comment:
```suggestion
                cur.commit()
```

?

Reply:
`conn.commit` does work -- I don't think you commit a cursor.